product_total_weight_from_packaging/__manifest__.py

```diff
@@ -0,0 +1,14 @@
+# Copyright 2020 Camptocamp SA
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl)
+{
+    "name": "Product Total Weight From Packaging",
+    "summary": "Compute estimated weight based on product's packaging weights",
+    "version": "18.0.1.0.0",
+    "development_status": "Beta",
+    "category": "Warehouse Management",
+    "website": "https://github.com/OCA/product-attribute",
+    "author": "Camptocamp, Odoo Community Association (OCA)",
+    "license": "AGPL-3",
+    "installable": True,
+    "depends": ["stock_packaging_calculator", "product_packaging_dimension"],
```

Comment:
Once https://github.com/OCA/stock-logistics-warehouse/pull/2173#issuecomment-2720351719 done

```suggestion
    "depends": ["product_packaging_calculator", "product_packaging_dimension"],
```

Reply:
PR here: https://github.com/OCA/product-attribute/pull/1932